### ISSUE TITLE
[Bug] Matching Target Characteristic for Current Characteristic Warning

### ISSUE THREAD

**Issue by mitch7391** [ztalbot2000/homebridge-cmd4#96]
**Describe The Bug:**

Hey John, I know you did some recent work around making sure all 'Current' characteristics (e.g. CurrentTemperature) has a matching 'Target' characteristic (e.g TargetTemperature). However in some cases like my temperature sensors, they do not have a characteristic for target. You can confirm that [here](https://developers.homebridge.io/#/service/TemperatureSensor). 

**To Reproduce:**

Restart Homebridge and the warnings will appear in the logs.

**Expected Behaviour:**

cmd4 should not be trying to make TargetTemperature for temperature sensors and the warnings should not be coming up.

**Paste of Logs:**
```
[27/02/2021, 08:35:32] [Cmd4] Warning, With fetch set to "Always" and polling for "CurrentTemperature" requested, you also must do polling of "TargetTemperature" or things will not function properly
```

**Environment:**

* **Node.js Version**: v14.16.0
* **NPM Version**: v7.5.6
* **Homebridge Version**: v1.3.1
* **homebridge-cmd4 Version**: v3.2.2
* **Operating System**: Raspbian
* **Process Supervisor**: hb-service

**Comment by ztalbot2000** [comment]
Hey Mitch,

If you go from the Home app and set the temperature, What is sent is by the
home app is to set TargetTemperature.  Not current temperature.

Ooh, maybe a sensor is different.  There it only asks for
currentTemperature and then the log does not apply.  Ok I can fix that.
Neede to check device type for warning message.

TTFN,
John

**Comment by mitch7391** [comment]
> If you go from the Home app and set the temperature, What is sent is by the
home app is to set TargetTemperature.  Not current temperature.

Yep that makes sense.

> Ooh, maybe a sensor is different.  There it only asks for
currentTemperature and then the log does not apply. 

Spot on the money :) I am not sure what other accessories might not need a 'target', but it might be worth checking all sensor type services for other people's accessories too.

**Comment by ztalbot2000** [comment]
Working on it.  The related characteristic must be in the same required or
optional characteristics of the device.

TTFN,
John

**Comment by ztalbot2000** [comment]
released as part of v3.3.0
